src/lib/swapper/swappers/ThorchainSwapper/utils/getQuote/getQuote.ts

```diff
@@ -101,3 +104,23 @@ export const getQuote = async ({
     return Ok(data)
   }
 }
+
+export const getQuote = async (
+  input: GetQuoteArgs,
+): Promise<Result<ThornodeQuoteResponseSuccess, SwapErrorRight>> => {
+  const initialQuoteResult = await _getQuote(input)
+
+  if (initialQuoteResult.isErr()) return initialQuoteResult
+
+  const initialQuote = initialQuoteResult.unwrap()
+
+  // refetch quote without affiliate fee if it's less than the thorchain outbound fee
+  if (
+    input.affiliateBps !== undefined &&
+    initialQuote.fees.affiliate < THORCHAIN_OUTBOUND_FEE_RUNE
+  ) {
+    return _getQuote({ ...input, affiliateBps: undefined })
```

Comment:
Wondering if we should be explicit here, as we specifically want `0`:

```suggestion
    return _getQuote({ ...input, affiliateBps: '0' })
```